frontend/mr-admin-flate/src/components/avtaler/AvtaleSkjemaContainer.tsx

```diff
@@ -79,13 +79,14 @@ export function AvtaleSkjemaContainer({
       administratorer: data.administratorer,
       avtaletype: data.avtaletype,
       prisbetingelser: erAnskaffetTiltak(data.tiltakstype.arenaKode)
-        ? data.prisbetingelser || null
+        ? data.prisbetingelser ?? null
         : null,
       beskrivelse: data.beskrivelse,
       faneinnhold: data.faneinnhold,
       personopplysninger: data.personvernBekreftet ? data.personopplysninger : [],
       personvernBekreftet: data.personvernBekreftet,
-      nusData: data.nusData || null,
+      nusData: data.nusData ?? null,
+      gruppeAmoKategori: data.gruppeAmoKategori ?? null,
```

Comment:
Gjelder ikke disse kategoriene også for Gruppe fag og yrkesopplæring? De skal vel også gjelde for individuelle opplæringstiltak på sikt. Kunne vi funnet på en navn som ikke nevner `gruppeAmo`?

Kanskje noe som bare nevner `opplaering`, hvis det er dette som er fellesnevneren for tiltakene som har denne ekstra statistikk-dataen :thinking:

Reply:
Godt poeng. Men de gjelder vel ikke for grufag??

Reply:
amo_kategorisering tror jeg gir mer mening